sig-architecture/production-readiness.md

```diff
@@ -28,6 +27,19 @@ aspects of the process.
     happens if it is subsequently upgraded again?
   - Are there tests for this?
 * Scalability
```

Comment:
Given that significant part of regressions is caused by expensive LIST calls, can we add explicit question about this? Like "Does enabling/using the feature will result in any new LIST calls? Are they one off or periodic ones?"

Reply:
The first question is addressing those - LIST are API calls and the first question basically forces you to mention them.

Reply:
Made that a bit more explicit now.